os_stub/spdm_device_secret_lib_sample/lib.c

```diff
@@ -28,6 +28,789 @@
 #include "library/memlib.h"
 #include "spdm_device_secret_lib_internal.h"
 
+#ifndef LIBSPDM_PRIVATE_KEY_USE_PEM
+#define LIBSPDM_PRIVATE_KEY_USE_PEM 0
+#endif
+
+#if !LIBSPDM_PRIVATE_KEY_USE_PEM
+
+/*ecp256 key for responder: https://lapo.it/asn1js/#MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgA-LbmJWXV0yozX6-mbTl9gXB8CC340M4Dzyh9fm-ymahRANCAATbwrK3gzzIheQ94fO64vKQjjAlFOH3qYIp2512L4ARMu6r4mjRIue9tHEnyHn73HyeM6ZnwhBHNjLFoaprK6rJ */
```

Comment:
please use LIBSPDM_RSA_SSA_SUPPORT, LIBSPDM_RSA_PSS_SUPPORT, LIBSPDM_ECDSA_SUPPORT to conditional include those features.

Reply:
Thanks. I have added the `LIBSPDM_RSA_SSA_SUPPORT, LIBSPDM_RSA_PSS_SUPPORT, LIBSPDM_ECDSA_SUPPORT` to make the code selectively include.